core/src/commonMain/kotlin/com/sourcepoint/mobile_core/network/Json.kt

```diff
@@ -16,3 +18,7 @@ val jsonWithNulls = Json {
     ignoreUnknownKeys = true
     explicitNulls = true
 }
+
+fun StringToJsonObject(str: String): JsonObject {
+    return decodeFromString<JsonObject>(str)
+}
```

Comment:
@Nevazhnovu I think we can rewrite this function a bit shorter (and starting with a lowcase letter)
```kotlin
fun stringToJsonObject(str: String): JsonObject = decodeFromString(str)
```

Or, maybe better, extend the String function:
```kotlin
fun String.encodeToJsonObject() = decodeFromString<JsonObject>(this)
```
So we can call it like:
```kotlin
"".encodeToJsonObject()
```

Reply:
I'll check if extension like this can be called from the native ios SDK